quintilesims/layer0
# 549: Fix API service LB health check for v0.10.6

**What does this pull request do?**

When running `l0-setup` apply for a new instance, after the resources are created, the health check was not succeeding and timing out. This happened because l0-setup tries to hit /health while the API was coded to have the health endpoint on /admin/health.

This change was originally implemented in api-refactor branches, so we are applying the /admin/helth -> /health change to the master/develop branches

**How should this be tested?**
Create a new l0 instance with `l0-setup` via both `init` and `apply`. The health check at the end of apply should succeed. All unit tests should pass.

closes #549
